requirements_detector/detect.py

```diff
@@ -2,7 +2,7 @@
 from pathlib import Path
 from typing import List, Union
 
-import toml
+import tomllib
```

Comment:
Hi, I just arrived here from encountering and then wanting to solve the same problem 😄 I see you got here first, but your solution is not quite right.

tomllib is only available in the standard library for python >=3.11.

Since this project declares compatibility with python >=3.7, you also need to add a dependency on [tomli](https://pypi.org/project/tomli/). It's exactly the same package as tomlib for python <3.11 


You'll need to do something like this for full compatibility: https://github.com/nat-n/poethepoet/blob/903ed9af0dd313a6da32f24ec4357101435976d8/poethepoet/config.py#L5